modulemd/v2/include/modulemd-2.0/modulemd-module-stream.h

```diff
@@ -395,4 +395,16 @@ gboolean
 modulemd_module_stream_build_depends_on_stream (ModulemdModuleStream *self,
                                                 const gchar *module_name,
                                                 const gchar *stream_name);
+
+/**
+ *modulemd_module_stream_equals:
```

Comment:
You need to leave a space between the asterisk (*) and the content of the comment or the documentation parser will have problems.